inc/classes/subscriber/preload/class-preload-subscriber.php

```diff
@@ -270,13 +283,15 @@ public function notice_preload_running() {
 			return;
 		}
 
-		$running = $this->homepage_preloader->get_number_of_preloaded_items();
+		$homepage_count = $this->homepage_preloader->get_number_of_preloaded_items();
+		$sitemap_count  = $this->sitemap_preloader->get_number_of_preloaded_items();
```

Comment:
Use the transients directly instead to simplify this. Then there's no need to inject the sitemap preloader.